requirements.txt

```diff
@@ -19,3 +19,4 @@ ninja
 packaging
 optree>=0.13.0
 cmake
+stdlib_list
```

Comment:
I thought we weren't going to take a dependency on stdlib_list?

Reply:
Oops forgot to remove, updated.